spec/jobs/searchgov_url_fetcher_job.rb

```diff
@@ -1,7 +1,7 @@
 require 'spec_helper'
 
 describe SearchgovUrlFetcherJob do
-  subject(:perform) { SearchgovUrlFetcherJob.perform_now(args[:searchgov_url]) }
+  subject(:perform) { SearchgovUrlFetcherJob.perform_now(searchgov_url) }
   let!(:searchgov_domain) { SearchgovDomain.create(domain: 'agency.gov', status: '200') }
```

Comment:
I don't see this variable used anywhere...?

Reply:
I've removed this variable.  It seemed to be necessary before when I was trying to test the fetching of a record, but now that I've changed that to what you recommended it isn't needed anymore.